### PR TITLE
rtt: set/restore scissor during rtt pass

### PR DESCRIPTION
**Pull request by jmoguillansky-gpsw**

### CODE REVIEW

File: libnodegl/node_rtt.c [hunk 1]
```diff
@@ -414,6 +414,12 @@ static void rtt_draw(struct ngl_node *node)
     const int vp[4] = {0, 0, s->width, s->height};
     ngli_gctx_set_viewport(gctx, vp);
 
+    int prev_scissor[4] = { 0 };
```
Comment (mbouron-gpsw) [inline libnodegl/node_rtt.c:417]: `int prev_scissor[4] = { 0 };` -> `int prev_scissor[4] = {0};`

File: libnodegl/node_rtt.c [hunk 2]
```diff
@@ -414,6 +414,12 @@ static void rtt_draw(struct ngl_node *node)
...
+    ngli_gctx_get_scissor(gctx, prev_scissor);
+
+    const int scissor[4] = { 0, 0, s->width, s->height };
```
Comment (mbouron-gpsw) [inline libnodegl/node_rtt.c:420]: Same as above.

**Review by mbouron-gpsw — COMMENTED** [review]
Other than remarks, LGTM.

File: libnodegl/node_rtt.c (hunk 2 above)
Comment (jmoguillansky-gpsw) [reply libnodegl/node_rtt.c]: updated

File: libnodegl/node_rtt.c (hunk 1 above)
Comment (jmoguillansky-gpsw) [reply libnodegl/node_rtt.c]: updated

**Review by wmacdonald-gpsw — APPROVED** [review]

**Review by mbouron-gpsw — APPROVED** [review]

**Pull request merged**